lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -636,6 +636,16 @@ def write(chunk)
                           record.delete(@trace_key)
                         end
 
+          ts_secs = begin
+                      Integer ts_secs
+                    rescue ArgumentError, TypeError
+                      ts_secs
+                    end
+          ts_nanos = begin
+                       Integer ts_nanos
+                     rescue ArgumentError, TypeError
+                       ts_nanos
```

Comment:
Ah, I see. That's probably me who added that test to be explicit. The downstream Logging API should convert it to `0` anyway since it's ultimately stored as protos. But we will leave it to the downstream component to decide.